README.md

```diff
@@ -1,26 +1,31 @@
 # KlaviyoSwift
 
 [![CI Status](https://travis-ci.org/klaviyo/klaviyo-swift-sdk.svg?branch=master)](https://travis-ci.org/klaviyo/klaviyo-swift-sdk)
+[![Swift](https://img.shields.io/badge/Swift-5.3_5.4_5.5_5.6-orange?style=flat-square)](https://img.shields.io/badge/Swift-5.3_5.4_5.5_5.6-Orange?style=flat-square)
+[![Swift Package Manager](https://img.shields.io/badge/Swift_Package_Manager-compatible-orange?style=flat-square)](https://img.shields.io/badge/Swift_Package_Manager-compatible-orange?style=flat-square)
 [![Version](https://img.shields.io/cocoapods/v/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
 [![License](https://img.shields.io/cocoapods/l/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
 [![Platform](https://img.shields.io/cocoapods/p/KlaviyoSwift.svg?style=flat)](http://cocoapods.org/pods/KlaviyoSwift)
 
-## Overview
 
-KlaviyoSwift is an SDK, written in Swift, for users to incorporate Klaviyo's event tracking functionality into iOS applications. We also provide an SDK written in [Objective-C](https://github.com/klaviyo/klaviyo-objc-sdk). The two SDKs are identical in their tracking functionality. **However, the KlaviyoSwift SDK is the only one that supports push notifications**. We strongly encourage the use of the KlaviyoSwift SDK.
+## Overview
 
-## Requirements
-- iOS 9.0+ 
-- Xcode 10.2+
-- Swift 5.0+
+KlaviyoSwift is an SDK, written in Swift that can be integrated into your iOS App. This will allow you to message your users via receive push notificaitons from Klaviyo. In addition you will be able to take advanatage of Klavaiyo's identificaiton and event tracking funtionality. Once integrated your marketing will be able to better understand your app users needs and send them timely messages via APNS.
```

Comment:
`This will allow you to message your users via receive push notificaitons from Klaviyo` should probably be `This will allow you to message your users via push notificaitons from Klaviyo`

Reply:
`push notifications`

Reply:
mm overall a bunch of spelling issues in this paragraph, double check it

Reply:
Fixed the typos in the following: 

KlaviyoSwift is an SDK, written in Swift that can be integrated into your iOS App. This will allow you to message your users via push notifications from Klaviyo. In addition you will be able to take advantage of Klaviyo's identification and event tracking functionality. Once integrated, your marketing team will be able to better understand your app users' needs and send them timely messages via APNS.

Reply:
Thanks!